src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,578 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+// import { Padding } from 'styled-components-spacing'
+import ReactMarkdown from 'react-markdown'
+// import generate from 'shortid'
+
+import { Grid, Row, Col } from '../../components/grid'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import Layout from '../../components/layout'
+// import { makeText } from '../../utils/makeText'
+import Head from '../../components/Common/Head'
+// import Image from '../../components/Common/Image'
+import VideoSection from '../../components/Common/VideoSection'
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+// import Statement from '../../components/Common/Statement'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+// const PaddingBottomGrid = styled(Grid)`
+//   padding-bottom: ${({ theme }) => theme.space[7]};
+// `
+
+const BlockRow = styled(Row)`
+  padding-top: ${({ theme, topIndex }) =>
+    topIndex ? theme.space[topIndex] : null};
+  padding-bottom: ${({ theme, bottomIndex }) =>
+    bottomIndex ? theme.space[bottomIndex] : null};
+  flex-direction: ${({ rowReverse }) => `row${rowReverse ? '-reverse' : null}`};
+  align-items: ${({ alignCenter }) => (alignCenter ? 'center' : null)};
+  width: ${({ widthPerc }) => widthPerc};
+
+/* ${breakpoint('smallTablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+    padding-bottom: ${({ theme }) => theme.space[6]};
+`}
+
+${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[7]};
+    padding-bottom: ${({ theme }) => theme.space[7]};
+`}
+
+${breakpoint('desktop')`
+    padding-bottom: ${({ theme }) => theme.space[8]};
+`} */
+`
+
+const renderImage = imageUrl =>
+  imageUrl && <img src={imageUrl} alt={imageUrl} />
```

Comment:
You'll need to use `<Image/>` from the common components. It's doing everything we already have defined here and also means we can use fluid image types from gatsby that we were discussing yesterday: https://using-gatsby-image.gatsbyjs.org

Please have a look at other places where we're using the gatsby image component, for example the homepage where we're using the `GatsbyContentfulFluid_withWebp` fragment. 

Performance is really important on this site, if we use a normal native HTML `<img />` tag this ends up with the page loading all of the images at once. Using the gatsby-image module, `Img` have lazy loading out of the box.

Reply:
Compare the two here in terms of resources size, 134kb compared to  3.4mb 😱 

Gatsby Image
![image](https://user-images.githubusercontent.com/7108120/60321915-badf4900-99b0-11e9-9b4c-a372f3d3aea7.png)

Native Img
![image](https://user-images.githubusercontent.com/7108120/60322045-14477800-99b1-11e9-949d-947415c07d41.png)

Reply:
Oh yes. You mentioned it yesterday and I forgot about it. Thanks for solving it 👍 Huge difference. Being lazy is the right way 😆